src/main/java/com/cinema/shop/model/Order.java

```diff
@@ -0,0 +1,68 @@
+package com.cinema.shop.model;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToMany
+    private List<Ticket> tickets;
+    private LocalDateTime orderDate;
+    @ManyToOne
+    private User user;
+
+    public Order() {
+    }
```

Comment:
do we need to write this explicitly?

Reply:
Hibernate creates it by default actually, so it's not really necessary, only if we want a constructor with arguments